components/device-mgt/org.wso2.carbon.device.mgt.core/pom.xml

```diff
@@ -222,6 +222,14 @@
             <artifactId>testng</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-module-testng</artifactId>
```

Comment:
Shall we make the scope of the test dependencies as "test"

Reply:
This is fixed in https://github.com/wso2/carbon-device-mgt/commit/6bfce09159ce282613c51f4c380f2b6ae0f222ac